dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2295,6 +2313,98 @@
         }
       }
     },
+    "ZeroShotClassificationTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a Zero Shot Classification task.",
+      "properties": {
+        "classificationType": {
+          "type": "string",
+          "description": "Specifies either one or multiple categories per document. Defaults to multi classification which may return more than one class for each document.",
+          "default": "Multi",
+          "enum": [
+            "Single",
+            "Multi"
+          ],
+          "x-ms-enum": {
+            "name": "ClassificationType",
+            "modelAsString": true
+          }
+        },
+        "categories" : {
+          "type": "array",
+          "description": "a list of categories to which input is classified to.",
```

Comment:
Are the categories known or can they be custom?

Reply:
yes they can be custom

Reply:
How would a user specify a custom category?

Reply:
For this type of classification categories are an input to the model, you can use any category you want (the model understands the category name and uses it for classification).

Reply:
Gotcha - so you don't need to train a model for this type of analysis? I guess I expected to see a project/deployment name in the input.

Reply:
Are the categories case-sensitive? Also how would I specify something with two words like "world politics"? Would "worldpolitics" work or does it need to be pascal case? It might be worth writing expected format into the description